demo-simple/src/main/java/com/novoda/downloadmanager/demo/MainActivity.java

```diff
@@ -72,15 +73,8 @@ public void onClick(View view) {
         buttonMigrate.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                migrationServiceBinder.bind();
-            }
-        });
-
-        View buttonAbortMigration = findViewById(R.id.button_abort_migration);
-        buttonAbortMigration.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View view) {
-                migrationServiceBinder.unbind();
+                downloadMigrationService.startMigration()
```

Comment:
We have an override to which a client can pass `Notification` customisations to `startMigration`.

Reply:
NIT: overload 🙃

Reply:
I have no idea what you are talking about :trollface:

Reply:
I *really* have no idea what you guys are talking about.